### PR TITLE
3.0 preview dev

### PR DESCRIPTION
**Pull request by notwaldorf**

### CODE REVIEW

File: paper-swatch-picker.js [hunk 1]
```diff
@@ -18,39 +30,26 @@ Example:
...
 
 ### Styling
 
 The following custom properties and mixins are available for styling:
 
 Custom property | Description | Default
 ----------------|-------------|----------
-`--paper-swatch-picker-color-size` | The size of each of the color boxes | `20px`
+`--paper-swatch-picker-color-size` | The size of each of the color boxes |
+`20px`
```
Comment (bicknellr) [inline paper-swatch-picker.js]: This table was broken by the formatter - you can prevent webmat from changing comments with [this](https://github.com/PolymerElements/iron-fit-behavior/pull/93#discussion_r199271183).

File: paper-swatch-picker.js [hunk 2]
```diff
@@ -18,39 +30,26 @@ Example:
...
 
 You can configure the color palette being used using the `colorList` array and
-the `columnCount` property, which specifies how many "generic" colours (i.e. columns
-in the picker) you want to display.
+the `columnCount` property, which specifies how many "generic" colours (i.e.
+columns in the picker) you want to display.
 
-    <paper-swatch-picker column-count=5 color-list='["#65a5f2", "#83be54", "#f0d551", "#e5943c", "#a96ddb"]'></paper-swatch-picker>
+    <paper-swatch-picker column-count=5 color-list='["#65a5f2", "#83be54",
+"#f0d551", "#e5943c", "#a96ddb"]'></paper-swatch-picker>
```
Comment (bicknellr) [inline paper-swatch-picker.js]: This seems to be broken by the formatter.

File: paper-swatch-picker-icon.js [hunk 3]
```diff
@@ -1,28 +1,24 @@
...
-*/
-/*
-  FIXME(polymer-modulizer): the above comments were extracted
-  from HTML and may be out of place here. Review them and
-  then delete this comment!
-*/
-;
+</iron-iconset-svg>
+`;
+template.setAttribute('style', 'display: none;');
```
Comment (bicknellr) [inline paper-swatch-picker-icon.js]: Do we actually need to set `display: none;` on templates? I've been removing it from the packages I've been working on.

Comment (notwaldorf) [reply paper-swatch-picker-icon.js]: i was afraid to

**Pull request merged**